d3/query-graphs.js

```diff
@@ -592,6 +594,47 @@ function drawQueryTree(target, treeData) {
         update(d);
     }
 
+    // Dash tween to make the highlighted edges animate from start node to end node
+    var tweenDash = function() {
+        var l = this.getTotalLength();
+        var i = d3.interpolateString("0," + l, l + "," + l);
+        return function(t) {
+            return i(t);
+        };
+    };
+
+    // Transition used to highlight edges on mouseover
```

Comment:
Personally, I am not a huge fan of mouse-hover transitions.
They don't help me to understand the tree in an easier way but they force me to wait 0.75 seconds between hovering a link and seeing the result of that hover

But probably that is only my personal opinion, so feel free to ignore this comment

Reply:
Just to clarify: highlighting through mouse-hover is fine
I just don't like the animation delay

Reply:
I also find the delay annoying. Will likely keep some form of hover highlighting, but not delayed.